db/seeds.rb

```diff
@@ -12,7 +12,7 @@
       random_level_count = Random.rand(level_count)
       random_event_type_count.times do |type_id|
         event_type = app.event_type "seed-type-#{type_id}"
-        event_type.levels [0..random_level_count].map do |level_id|
+        event_type.levels [0...random_level_count].map do |level_id|
```

Comment:
`[0...random_level_count]` creates an array containing range from `0` to `random_level_count` (non inclusive). Is that really what you wanted?